src/test/kotlin/uk/gov/gdx/datashare/controller/EventsControllerTest.kt

```diff
@@ -166,36 +168,6 @@ class EventsControllerTest {
     }
   }
 
-  @Test
```

Comment:
Removing this test seems a little orthogonal to the actual change, is there a reason it doesn't work?

Reply:
Its a duplicate test, it broke when I was doing this change which is why I noticed it